Dockerfile

```diff
@@ -1,26 +1,7 @@
-FROM node:10-alpine
+FROM node:12
 
 COPY . /code
 WORKDIR /code
 
 RUN npm i
-
-# Install Chrome
-# Add chromium and all necessary tools to run chrome headless browser (used for front-end testing)
-RUN apk -U --no-cache \
-	--allow-untrusted add \
-    zlib-dev \
-    chromium \
-    xvfb \
-    wait4ports \
-    xorg-server \
-    dbus \
-    ttf-freefont \
-    mesa-dri-swrast \
-    grep \
-    udev
-
-ENV CHROME_BIN=/usr/bin/chromium-browser
-ENV CHROME_PATH=/usr/lib/chromium/
-
-CMD ["npm run bootstrap && npm run build"]
\ No newline at end of file
+CMD npm start
```

Comment:
Enkel een `npm start` zal vanuit een cleane codebase niet lukken want op dat moment zijn de packages niet gebuild en krijg je bijvoorbeeld deze error: 
```
Could not resolve @acpaas-ui/ngx-calendar relative to [object Object]..
```

Je moet er vanuit gaan dat een docker image zichzelf volledig zelf kan opbouwen. Als jij toevallig lokaal al een build hebt van de packages, dan zal die de dist folder mee kopieren waardoor het wel werkt, maar eigenlijk zou die docker file alles zelf moeten builden. 

M.a.w. er ontbreekt nog een `npm run build:all` in het script.

Reply:
Nog een tweede bemerking, ik weet niet direct waarvoor deze dockerfile allemaal gebruikt wordt. Indien dat enkel voor lokaal development is, is er geen probleem maar dan zou ik de naam misschien aanpassen naar `Dockerfile.dev` waardoor dat iets explicieter gemaakt wordt. 

De docker image gebruiken om de style guide te server is op deze manier alleszins niet de bedoeling. `npm run start` gaat een dev server opstarten. Voor echte online omgevingen is het de bedoeling dat de styleguide gebuild wordt en dan gehost via nginx of apache in de docker image (multi-stage).

Reply:
Als je angular runt vanuit docker moet je ook een ip adres binden. Anders laat de dev server het niet toe om die te accessen. 
`ng serve styleguide --host=0.0.0.0`